code/ship/ship.cpp

```diff
@@ -3561,6 +3575,12 @@ static int parse_ship_values(ship_info* sip, const bool is_template, const bool
 	if(optional_string("$Scan range Capital:"))
 		stuff_float(&sip->scan_range_capital);
 
+	if (optional_string("$Ask Help Shield Percent:"))
+		stuff_float(&sip->ask_help_shield_percent);
+
+	if (optional_string("$Ask Help Hull Percent:"))
+		stuff_float(&sip->ask_help_hull_percent);
```

Comment:
Both these cases need some error handling to make sure invalid values do not break the code.

Reply:
Will do.